tests/lib/cli.js

```diff
@@ -633,6 +633,30 @@ describe("cli", function() {
         });
     });
 
+    describe("when passed --no-inline-config", function() {
+
+        it("should pass allowInlineConfig:false to CLIEngine and ignore inline config when executing on text", function() {
```

Comment:
This isn't actually what the test is doing. Looks like previous changes were reverted?